store/tikv/2pc.go

```diff
@@ -402,9 +402,23 @@ func (c *twoPhaseCommitter) prewriteSingleBatch(bo *Backoffer, batch batchKeys)
 	}
 }
 
+func getTxnPriority(txn *tikvTxn) pb.CommandPri {
+	if pri := txn.us.GetOption(kv.Priority); pri != nil {
+		switch pri.(int) {
+		case kv.PriorityNormal:
```

Comment:
This will be called by every kv request. Without this branch is still clear enough.